calypso/api.go

```diff
@@ -87,16 +90,34 @@ func (c *Client) CreateLTS(ltsRoster *onet.Roster, darcID darc.ID, signers []dar
 	return reply, nil
 }
 
-// Authorise adds a ByzCoinID to the list of authorized IDs for each
-// server in the roster. The AuthoriseByzcoinID service refuses requests
-// that do not come from localhost.
+// Authorise - deprecated, please see Authorize
```

Comment:
```
Authorise blabla...
Deprecated: please see Authorize
```

Reply:
In order to have a tag to find deprecated code later

Reply:
Thanks for catching up with me...